src/Microsoft.FeatureManagement/FeatureManagementError.cs

```diff
@@ -9,18 +9,23 @@ namespace Microsoft.FeatureManagement
     public enum FeatureManagementError
     {
         /// <summary>
-        /// A feature filter that was listed for feature evaluation was not found.
+        /// A feature filter configured for the feature being evaluated is an ambiguous reference to multiple registered feature filters.
         /// </summary>
-        MissingFeatureFilter,
+        AmbiguousFeatureFilter,
```

Comment:
Be careful about changing the order of enums. They have underlying numeric values. If someone was serializing these errors as number, this would break.

Reply:
Undid the re-ordering